src/Facebook/FacebookClient.php

```diff
@@ -186,6 +195,21 @@ public function prepareRequestMessage(FacebookRequest $request)
         ];
     }
 
+    public function setDefaultRequestTimeout(int $timeout): void
+    {
+        $this->default_request_timeout = $timeout;
+    }
+
+    public function setFileUploadRequestTimeout(int $timeout): void
+    {
+        $this->file_upload_request_timeout = $timeout;
+    }
+
+    public function setVideoUploadRequestTimeout(int $timeout): void
+    {
+        $this->video_upload_request_timeout = $timeout;
+    }
```

Comment:
Maybe we should consider removing these setters, and instead pass the values into the constructor with the defaults as above.

Reply:
That could work. The reason I held off on it was that FacebookClient is used a fair amount of places and I didn't want to change the instantiations. But looking at it we could provide default arguments in the constructor and it'd be the same..